test/utils/fizzy_engine.cpp

```diff
@@ -100,7 +100,6 @@ WasmEngine::Result FizzyEngine::execute(
 {
     const auto [trapped, result_stack] =
         fizzy::execute(*m_instance, static_cast<uint32_t>(func_ref), args);
-    assert(result_stack.size() <= 1);
```

Comment:
This just shows fizzy's "public api" is broken right now :)

Hopefully after the span things are going in, we can merge a version of #219 can go in and fix this.

Reply:
I was confused at first, but truly we dump whole stack also in the case when returning `trapped`.
I can add disabled test for such case, but I don't think this really helps.